app/views/Settings/Section.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
This file looks a little complicated for what it does. You could likely inline a lot of this and handle the layout purely with styles.